src/sentry/static/sentry/app/views/organizationDiscover/missingProjectWarningModal.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Modal, {Header, Body, Footer} from 'react-bootstrap/lib/Modal';
+import Button from 'app/components/button';
+import SentryTypes from 'app/sentryTypes';
+import {t} from 'app/locale';
+
+export default class MissingProjectWarningModal extends React.Component {
+  static propTypes = {
+    organization: SentryTypes.Organization.isRequired,
+    projects: PropTypes.arrayOf(PropTypes.number).isRequired,
+    closeModal: PropTypes.func,
+  };
+
+  renderProject(id) {
+    id = id.toString();
```

Comment:
IMO shadowing params 🙅‍♂️

Reply:
We reassign variables like this this fairly extensively in the application, but will update